pinc/quizzes.inc

```diff
@@ -111,7 +111,7 @@ new QuizLevel(
     "proof",
     "<p>" . _("This section covers topics that are not addressed in the <a href='../faq/proofreading_guidelines.php'>proofreading guidelines</a>. The pages include items that you are likely to see when proofreading HARD (and some Average) projects.") .
         "</p>\n<p>" .
-        _("If you are not yet familiar with a topic, you can learn about it by using the tutorial mode or following the \"Information\" link for that topic.") .
+        _("The following quizzes are not required, but can be useful. If you are not yet familiar with a topic, you can learn about it by using the tutorial mode or following the \"Information\" link for that topic.") .
```

Comment:
This probably belongs in `pgdp-production` and not `master` because any sort of quiz *requirement* is a local hack (see https://github.com/DistributedProofreaders/dproofreaders/commit/6396a28791dba4f88e200e46e5938ae0727311c2 in `pgdp-production`).

Reply:
Oops, I totally forgot to take that into account, thanks. Now to figure out how to split the commits into separate branches, and get things straightened out.

Reply:
Done.